cmd/kube-proxy/app/server.go

```diff
@@ -147,7 +147,9 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.ConfigFile, "config", o.ConfigFile, "The path to the configuration file.")
 	fs.StringVar(&o.WriteConfigTo, "write-config-to", o.WriteConfigTo, "If set, write the default configuration values to this file and exit.")
 	fs.StringVar(&o.config.ClientConnection.Kubeconfig, "kubeconfig", o.config.ClientConnection.Kubeconfig, "Path to kubeconfig file with authorization information (the master location can be overridden by the master flag).")
-	fs.StringVar(&o.config.ClusterCIDR, "cluster-cidr", o.config.ClusterCIDR, "The CIDR range of pods in the cluster. When configured, traffic sent to a Service cluster IP from outside this range will be masqueraded and traffic sent from pods to an external LoadBalancer IP will be directed to the respective cluster IP instead")
+	fs.StringVar(&o.config.ClusterCIDR, "cluster-cidr", o.config.ClusterCIDR, "The CIDR range of pods in the cluster. When configured, traffic sent to a Service cluster IP from outside this range will be masqueraded and traffic sent "+
+		"from pods to an external LoadBalancer IP will be directed to the respective cluster IP instead. "+
+		"For dual-stack clusters, a comma-separated list is accepted with at least one CIDR per IP family (IPv4 and IPv6).")
```

Comment:
@tyler-lloyd I believe it's actually zero-or-one per family currently. You can't give it two v4 or two v6

Reply:
you're right. I'll update the comment.

Reply:
@dcbw let me know what you think of the wording now.

Reply:
you can find the validation logic here with the exact details https://github.com/kubernetes/kubernetes/blob/de52812264b60d57f6adb4c6965b8d8bd41418b9/pkg/proxy/apis/config/validation/validation.go#L77-L97

Reply:
@aojea - right, I think my updated comment should be sufficient here?

> For dual-stack clusters, a comma-separated list is accepted with exactly 1 CIDR from each IP family (IPv4 and IPv6).